R/markdown.R

```diff
@@ -87,5 +87,7 @@ markdown_text2 <- function(text, pkg, ...) {
     return(paste0(xml2::xml_contents(children), collapse=""))
   }
 
-  return(paste0(as.character(children), collapse=""))
+  output <- paste0(as.character(children), collapse="")
+
+  gsub("\\\n", "", output)
```

Comment:
pkgdown websites aren't minified but this should prevent maxified some parts, at least. :slightly_smiling_face: